com.github.sdbg.integration.jdt/src/com/github/sdbg/integration/jdt/JDTBreakpointPathResolver.java

```diff
@@ -2,6 +2,7 @@
 
 import com.github.sdbg.debug.core.breakpoints.IBreakpointPathResolver;
 
+import org.eclipse.core.resources.IResource;
```

Comment:
What problem is being fixed here, i.e. can you describe a use case where the current code does not really work?